source/enumerate-collections.rst

```diff
@@ -362,7 +368,8 @@ Example return (in PHP, but abbreviated)::
       [5] => class MongoCollection#11 { }
     }
 
-Drivers MUST specify the ``nameOnly`` option when executing the ``listCollections`` command for this method,
+Drivers MAY allow the ``nameOnly`` and ``authorizedCollections`` options
+to be passed when executing the ``listCollections`` command for this method,
```

Comment:
This is a behavioral change for `nameOnly`. I think you should retain MUST for that option since drivers should only need the name to construct their collection objects.

If anything, you can improve the original text by stating that drivers should specify `true` for `nameOnly`. "Drivers MUST specify the ``nameOnly`` option" is a bit vague, since it's only talking about the option being specified and says nothing of its value.

You can add a separate paragraph to state that drivers MAY support `authorizedCollections`, similar to the text used in the preceding section.

Reply:
done